libs/gl-client-py/glclient/glclient.pyi

```diff
@@ -15,6 +15,7 @@ class Signer:
     def version(self) -> str: ...
     def is_running(self) -> bool: ...
     def shutdown(self) -> None: ...
+    def create_rune(self) -> str: ...
```

Comment:
Seems this is missing, at least in this commit.

Reply:
Yes thanks for noticing! This is dead code from early experiments as we do not hand out runes so far.